projects/libs/core-library-typescript/test/gossip/gossip.spec.ts

```diff
@@ -74,7 +74,7 @@ describe("Gossip", function () {
 
   describe("with signatures", () => {
     before(async function() {
-      this.timeout(6000);
+      this.timeout(20000);
```

Comment:
huh?

Reply:
Sometimes it takes more times to generate keys and it fails for that reason.